lib/Runtime/Language/InterpreterStackFrame.cpp

```diff
@@ -8330,6 +8331,98 @@ const byte * InterpreterStackFrame::OP_ProfiledLoopBodyStart(const byte * ip)
         return args;
     }
 
+    HeapArgumentsObject * InterpreterStackFrame::CreateEmptyHeapArgumentsObject(ScriptContext* scriptContext)
+    {
+        HeapArgumentsObject * args = JavascriptOperators::CreateHeapArguments(this->function->GetRealFunctionObject(), this->m_inSlotsCount - 1, 0, nullptr, scriptContext);
+        this->m_arguments = args;
+        return args;
+    }
+
+    void InterpreterStackFrame::TrySetFrameObjectInHeapArgObj(ScriptContext * scriptContext, bool hasNonSimpleParams)
+    {
+        if (PHASE_OFF1(Js::StackArgFormalsOptPhase))
```

Comment:
Removed. Left overs.

---
In reply to: [67786183](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67786183) [](ancestors = 67786183)